src/index.ts

```diff
@@ -478,6 +477,7 @@ async function runCommand(
 	cmd: string,
 	argv: string[],
 ): Promise<void> {
+	// eslint-disable-next-line @typescript-eslint/no-var-requires
 	const script = require(path.join(__dirname, 'commands', cmd));
```

Comment:
It seems that we no longer use this function and we can drop it since there is only a dymmy command atm
See: https://github.com/balena-io/open-balena-api/pull/22/files